focus-block/src/Components/Landing/Landing.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component, Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import ReactGA from 'react-ga';
+
+class Landing extends Component {
+  render() {
+    ReactGA.initialize('UA-90746218-3');
+    ReactGA.pageview(window.location.pathname);
+
+    return (
+      <Fragment>
+        <div className="first-screen">
+          <div className="first-screen__triangles">
+            <div className="first-screen__triangle1"></div>
+            <div className="first-screen__triangle2"></div>
+            <div className="first-screen__triangle3"></div>
+          </div>
+          <div className="first-screen__logo">
+            <img src={require("../../Assets/focusblock-logo-white.svg")} alt="" />
```

Comment:
Make this a property for image please :)

Reply:
Duplicate of global "requirements should be at the top" comment.